.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Shouldn't this be a Biome config?

Reply:
hmm i wonder why my biome config is not catching it locally then